### PR TITLE
toggleButton

### PR DESCRIPTION
**Pull request by murliDN**

### CODE REVIEW

File: src/components/toggleButton.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import {View, TouchableOpacity, StyleSheet, Switch} from "react-native";
```
Comment (jtibrewal09) [inline src/components/toggleButton.js:2]: What's the use of touchableOpacity?

File: src/components/toggleButton.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+                    Colors.white
+                }
+                activeThumbColor={
+                    Colors.white
+                }
+                // for web
+                activeTrackColor={
+                    Colors.primary
+                }
+                // for web
```
Comment (jtibrewal09) [inline src/components/toggleButton.js:28]: Web means the mobile site?

File: src/components/toggleButton.js (hunk 1 above)
Comment (murliDN) [reply src/components/toggleButton.js:2]: will remove it

File: src/components/toggleButton.js (hunk 2 above)
Comment (murliDN) [reply src/components/toggleButton.js:28]: yes

File: src/screens/componentsList/index.js [hunk 3]
```diff
@@ -2,13 +2,19 @@ import { StatusBar } from "expo-status-bar";
...
 
 import * as Colors from "../../utils/colors";
+import invertValue from "../../utils/common"
 
 export default function ComponentList(props) {
     const [value, onChangeText] = React.useState('');
     const { navigation } = props;
+    const [isEnabled, setIsEnabled] = React.useState(false);
+    const toggleSwitch = (isEnabled) => {
+        setIsEnabled(isEnabled => !isEnabled)
```
Comment (jtibrewal09) [inline src/screens/componentsList/index.js]: This entire function can be written in a better way.

File: src/utils/colors.js [hunk 4]
```diff
@@ -1,3 +1,4 @@
 export const primary = "#9C7BFC";
 export const white = "#FFF";
 export const black = "#0E1B42";
+export const gray = "#d3d3d3"
```
Comment (jtibrewal09) [inline src/utils/colors.js:4]: Although, not mandatory. But, we should use semicolon at the end of line to avoid unnecessary errors.

**Pull request merged**